tests/chicory-tests/option-tests/README

```diff
@@ -1,3 +1,5 @@
 This directory tests Chicory command-line options.
 Each test runs Chicory and Daikon on the StackAr program, but each test
 uses a different set of command-line options.
+
+The test suite has been augmented to cover the addition of exception exit ppts.
```

Comment:
Delete this.  It isn't helpful in the code -- it's just a code review comment, and as such it doesn't belong in the code.